verify/Dockerfile

```diff
@@ -46,28 +46,41 @@ RUN dnf -y update && \
     dnf -y builddep /tmp/cockpit.spec && \
     dnf clean all
 
-RUN mkdir -p /usr/local/bin /home/user /build
-ADD cockpit-verify vm-prep network-cockpit.xml /usr/local/bin/
+RUN mkdir -p /usr/local/bin /home/user /build /build/tmp /secrets /images
+ADD cockpit-verify install-service vm-prep network-cockpit.xml /usr/local/bin/
 
-RUN chown -R user /build /home/user
 RUN usermod -a -G mock user
 RUN echo 'user ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers
 RUN echo "config_opts['basedir'] = '/build/mock/'" >> /etc/mock/site-defaults.cfg
+RUN printf '[user]\n\t\nemail = cockpituous@cockpit-project.org\n\tname = Cockpituous\n' >/home/user/.gitconfig && \
+    ln -s /images /build/images && \
+    mkdir -p /home/user/.config /home/user/.ssh && \
+    ln -snf /secrets/ssh-config /home/user/.ssh/config && \
+    ln -snf /secrets/github-token /home/user/.config/github-token && \
```

Comment:
Can these be copied at container start time instead of linked at container build time, to ensure that the container doesn't change them? (Like the chown below)